conf/environments.js

```diff
@@ -26,6 +26,16 @@ module.exports = {
             globalReturn: true
         }
     },
+    commonjs: {
+        globals: {
+            module: false,
+            require: false,
+            exports: false
+        },
+        ecmaFeatures: {
+            globalReturn: true
```

Comment:
For my understanding... Does this hint to ESLint that all code written should behave as if not in the global scope? That is, variables would shadow globals not overwrite them?
If not, what exactly does this setting do?